src/styles/styles.css

```diff
@@ -0,0 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Bitter&family=Montserrat&display=swap');
+
+html {
```

Comment:
Since you just started the implementation and it's still easy to switch: Have you heard of [styled-components](https://styled-components.com/)? It's a bit more modern way of styling with React imo. 

One advantage is if you need to control animations or dynamic styles with JS you can simply share variables between the styles and component code. That's not the case here but styled-components is anyway a great thing to put on your CV

Reply:
This is just a suggestion btw. I'll leave it up to you :wink:

Reply:
Oh I like this!
Do you still employ your own custom css file for changing the overall background color (or other items that affect the whole site) of the page?

Reply:
It's a great library :smile: You wouldn't use the `.css` file anymore but rather [global styles](https://styled-components.com/docs/api#createglobalstyle) for things like styles that affect the whole page and additionally a [theme](https://styled-components.com/docs/advanced#theming) to store global variables like sizes, colors etc.

Reply:
So as a change request move these styles to the global styles mentioned above. Feel free to create a theme as well. You can also do that in the next PR. I don't mind.

Not sure if `@import url` works with styled-components the same way. Otherwise you can move the import to index.html